styles/focus/catppuccin.user.css

```diff
@@ -0,0 +1,509 @@
+      }
+      .parent-menu-options-container {
+        background: @crust;
+        .parent-menu-option-title,
+        .parent-menu-sub-options-header {
+          background: @mantle;
+          color: @accent-color !important;
+        }
+        .parent-menu-option-title:hover,
+        .parent-menu-sub-options-header:hover {
+          background: fade(@mantle, 30%) !important;
+          color: @accent-color !important;
+        }
+      }
+      .parent-menu-sub-options {
+        background: fade(@mantle, 50%);
+        .parent-menu-form-option-title {
+          color: @accent-color !important;
+          &:hover {
+            color: fade(@accent-color, 75%) !important;
+          }
+        }
+      }
+      .parent-menu-footer,
+      .parent-menu-footer-button {
+        background: @surface0;
+        color: @accent-color !important;
+      }
+    }
+    .portal-block {
+      .colored-tables();
+      color: @text;
+      background: @surface1 !important;
+
+      .portal-block-title {
+        background: @accent-color;
+        color: @base;
+      }
+      .portal-block-subtitle {
+        background: @surface0;
+      }
+      .portal-background-color {
+        background: @accent-color;
+      }
+      .custom-background-color {
+        &:not(.active) {
+          background: @overlay0 !important;
+          color: @text !important;
+        }
+        &:is(.active) {
+          background: @accent-color !important;
+          color: @base !important;
+        }
+      }
+      .portal-block-tabs {
+        background: @surface1 !important;
+        color: @base;
+        .portal-block-tab {
+          &:not(.active) {
+            background: @overlay0 !important;
+            color: @text !important;
+          }
+          &:is(.active) {
+            background: @accent-color;
+            color: @base;
+          }
+        }
+      }
+      .portal-item-header {
+        background: @overlay1;
+        .portal-item-title,
+        .portal-item-subtitle,
+        time {
+          color: @text !important;
+        }
+      }
+      .portal-item-content {
+        background: @overlay0;
+      }
+      .portal-planner {
+        .portal-planner-date-text {
+          color: @text !important;
+        }
+        .portal-planner-date {
+          background: fade(@accent-color, 50%) !important;
+        }
+      }
+      .district-events-header {
+        background: @surface0;
+      }
+      .no-items {
+        color: @text;
+      }
+    }
+    .category-header {
+      color: @text;
+      background: @base !important;
+    }
+    .filter-container i {
+      color: @text !important;
+    }
+    .fields-column-expander {
+      background: @accent-color !important;
+      color: @base !important;
+      border-color: @overlay0 !important;
+    }
+    .ui.attached.segment {
+      border-color: @overlay0 !important;
+    }
+    .fields-column .info-segment {
+      * {
+        color: @text !important;
+      }
```

Comment:
Try to avoid the `*` selector. In these child elements, please find the CSS class that is actually changing the text color and set that instead.

Reply:
it uses inline css quite a lot

Reply:
I'd prefer something like:
```css
[style*="color: #000;"] {
  color: @text !important;
}
```

Reply:
that'd probably require writing 20 more selectors than is necessary.

Reply:
> and sometimes it doesnt bother styling the text (and relies on the UA's css to color it black)

So just put a top level `color: @text;`?

> it's just a whole mess but * seemed to work without a problem

Yes but it should be used sparingly. Can you give me examples of what other selectors there are? How many shades of black text or whatever are there on this site?

Reply:
ok so i just realized that you were looking at the table styles lol, for some reason i thought it was toplevel :skull:
i removed it and dont seem to notice any difference, so i'll remove it